modules/cli/src/main/scala/scala/cli/commands/util/BuildCommandHelpers.scala

```diff
@@ -15,5 +17,13 @@ trait BuildCommandHelpers { self: ScalaCommand[_] =>
         self.argvOpt.map(_.mkString(" ")).getOrElse(actualFullCommand),
         logger
       )
+
+    /** -O -d defaults to --compile-output; if both are defined, --compile-output takes precedence
+      */
+    def copyOutput(sharedOptions: SharedOptions): Unit =
+      sharedOptions.compilationOutput.filter(_.nonEmpty)
+        .orElse(sharedOptions.scalac.getForKey("-d").headOption)
+        .filter(_.nonEmpty)
+        .map(os.Path(_, Os.pwd)).foreach(output => os.copy.over(successfulBuild.output, output))
```

Comment:
If the output path doesn't exist, will it create directories recursively?

Reply:
good catch. should be fixed now, now it will.